<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenerateUmpireDialog</class>
 <widget class="QDialog" name="GenerateUmpireDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wygeneruj Sędziów</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Liczba sędziów do wygenerowania:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSpinBox" name="Ilosc">
       <property name="suffix">
        <string/>
       </property>
       <property name="prefix">
        <string/>
       </property>
       <property name="maximum">
        <number>99</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="displayIntegerBase">
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Konkurencje</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="SiatkowkaPlazowaG">
        <property name="text">
         <string>Siatkówka Plażowa Główny</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="SiatkowkaPlazowaP">
        <property name="text">
         <string>Siatkówka Plażowa Pomocniczy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="PrzeciaganieLinyG">
        <property name="text">
         <string>Przeciąganie Liny Główny</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="DwaOgnieG">
        <property name="text">
         <string>Dwa Ognie Główny</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="Uwaga">
     <property name="text">
      <string>Uwaga! Wygenerowane zostanie 0 sędziów.</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Anuluj">
       <property name="text">
        <string>Anuluj</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
